app/Http/Controllers/Controller.php

```diff
@@ -25,4 +26,10 @@ protected function getAppSecret()
     {
         return env('CLIENT_SECRET');
     }
+
+    protected function storeToSession(array $configs) {
```

Comment:
Do we know the structure of config? I guess it is a key value pair. In this case you could add

```php
/*
 * @param $configs array<string, mixed>
 */
```